server/routes/findInterventions/searchResultsPresenter.test.ts

```diff
@@ -0,0 +1,151 @@
+import { DeepPartial } from 'fishery'
+import SearchResultsPresenter from './searchResultsPresenter'
+import interventionFactory from '../../../testutils/factories/intervention'
+import eligibilityFactory from '../../../testutils/factories/eligibility'
+import { Intervention } from '../../services/interventionsService'
+
+describe(SearchResultsPresenter, () => {
+  describe('results', () => {
+    it('contains as many items as there are interventions', () => {
+      const presenter = new SearchResultsPresenter(interventionFactory.buildList(3))
+
+      expect(presenter.results.length).toBe(3)
+    })
+
+    function linesForKey(key: string, params: DeepPartial<Intervention>): string[] {
+      const presenter = new SearchResultsPresenter([interventionFactory.build(params)])
+      const item = presenter.results[0].summary.find(anItem => anItem.key === key)
+      if (item === undefined) {
+        fail(`Didn't find item for key ${key}`)
+      }
+      return item.lines
+    }
+
+    describe('Type', () => {
+      it('is “Dynamic Framework”', () => {
+        expect(linesForKey('Type', {})).toEqual(['Dynamic Framework'])
+      })
+    })
+
+    describe('Region', () => {
+      it('is “TODO”', () => {
```

Comment:
In the [designs](https://www.figma.com/file/hvnANWQPHEornIY2jFXocu/BETA-Single-role-user-for-dashboard%2Fintervention-progress?node-id=2173%3A28519), this is, for example, "North East" or "Yorkshire and the Humber". I don't think we have anything in the `Intervention` DTO that corresponds to these, which are perhaps the NPS regions?

Reply:
Ming's let me know that the design was from before we decided to use PCC regions, so I've removed the Region for now - am waiting for final clarification from Rhiannon/Tom but I think we can always add it back in when we know more.